### ISSUE TITLE
Block PVC are marked as FileSystemResizeRequired 

### ISSUE THREAD

**Issue by wavezhang** [ceph/ceph-csi#1384]
# Describe the bug #

Block PVC are marked as FileSystemResizeRequired 

# Environment details #

- Image/version of Ceph CSI driver : v1.2.1
- Helm chart version :
- Kernel version :
- Mounter used for mounting PVC (for cephfs its `fuse` or `kernel`. for rbd its
  `krbd` or `rbd-nbd`) :
- Kubernetes cluster version :
- Ceph cluster version :

# Steps to reproduce #

1. create a block pvc
2. update pvc size
1. See error

# Actual results #
```
Name:          block-pvc
Namespace:     default
StorageClass:  rook-ceph-block
Status:        Bound
Volume:        pvc-20e0a018-ff8f-4068-9fbf-8a714b17b656
Labels:        username.infra.genos.io=kubernetes-admin
Annotations:   kubectl.kubernetes.io/last-applied-configuration:
                 {"apiVersion":"v1","kind":"PersistentVolumeClaim","metadata":{"annotations":{},"name":"block-pvc","namespace":"default"},"spec":{"accessMo...
               pv.kubernetes.io/bind-completed: yes
               pv.kubernetes.io/bound-by-controller: yes
               volume.beta.kubernetes.io/storage-provisioner: rook-ceph.rbd.csi.ceph.com
Finalizers:    [kubernetes.io/pvc-protection]
Capacity:      1Gi
Access Modes:  RWO
VolumeMode:    Block
Mounted By:    <none>
Conditions:
  Type                      Status  LastProbeTime                     LastTransitionTime                Reason  Message
  ----                      ------  -----------------                 ------------------                ------  -------
  FileSystemResizePending   True    Mon, 01 Jan 0001 00:00:00 +0000   Wed, 03 Jun 2020 17:48:51 +0800           Waiting for user to (re-)start a pod to finish file system resize of volume on node.
Events:                     <none>
```
# Expected behavior #

Block PVC successfully resized with no FileSystemResizeRequired events.
 
# Logs #

If the issue is in PVC creation, deletion, cloning please attach complete logs
of below containers.

- csi-provisioner and csi-rbdplugin/csi-cephfsplugin container logs from the
  provisioner pod.

If the issue is in PVC resize please attach complete logs of below containers.

- csi-resizer and csi-rbdplugin/csi-cephfsplugin container logs from the
  provisioner pod.

If the issue is in snapshot creation and deletion please attach complete logs
of below containers.

- csi-snapshotter and csi-rbdplugin/csi-cephfsplugin container logs from the
  provisioner pod.

If the issue is in PVC mounting please attach complete logs of below containers.

- csi-rbdplugin/csi-cephfsplugin and driver-registrar container logs from
  plugin pod from the node where the mount is failing.

- if required attach dmesg logs.

**Note:-** If its a rbd issue please provide only rbd related logs, if its a
cephfs issue please provide cephfs logs.

# Additional context #

For example:

Any existing bug report which describe about the similar issue/behavior

**Comment by humblec** [comment]
@wavezhang regardless of that event, it should work. I mean you get the new size in the POD, isnt it ?

**Comment by wavezhang** [comment]
If I didnot bind pvc to any pod，what is expected is that pvc resized as soon as I update pvc size， and pvc capacity updated to the new size.  But actually pvc capacity not updated and confusing my upstream apps。

**Comment by nixpanic** [comment]
@humblec please close this as duplicate of #1379 if that is the case. Or add `Fixes: #1384` to the description of PR #1386.

**Comment by Madhu-1** [comment]
This is fixed already
